examples/with-gnosis/src/createNewWallet.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  TurnkeyClient,
+  createActivityPoller,
+  TurnkeyActivityError,
+} from "@turnkey/http";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
+import * as crypto from "crypto";
+import { refineNonNull } from "./util";
+
+export async function createNewWallet() {
+  console.log("creating a new wallet on Turnkey...\n");
+
+  const walletName = `ETH Wallet ${crypto.randomBytes(2).toString("hex")}`;
+
+  try {
+    const turnkeyClient = new TurnkeyClient(
+      { baseUrl: process.env.BASE_URL! },
+      new ApiKeyStamper({
+        apiPublicKey: process.env.API_PUBLIC_KEY!,
+        apiPrivateKey: process.env.API_PRIVATE_KEY!,
+      })
+    );
+
+    const activityPoller = createActivityPoller({
+      client: turnkeyClient,
+      requestFn: turnkeyClient.createWallet,
+    });
+
+    const completedActivity = await activityPoller({
+      type: "ACTIVITY_TYPE_CREATE_WALLET",
+      timestampMs: String(Date.now()),
+      organizationId: process.env.ORGANIZATION_ID!,
+      parameters: {
+        walletName,
+        accounts: [
+          {
+            curve: "CURVE_SECP256K1",
+            pathFormat: "PATH_FORMAT_BIP32",
+            path: "m/44'/60'/0'/0/0",
+            addressFormat: "ADDRESS_FORMAT_ETHEREUM",
+          },
+          {
+            curve: "CURVE_SECP256K1",
+            pathFormat: "PATH_FORMAT_BIP32",
+            path: "m/44'/60'/0'/0/1",
```

Comment:
nit: since this is supposed to demonstrate multisig - probably better to update the account index instead of address index, but not a huge deal.

```
path: "m/44'/60'/0'/0/0"
path: "m/44'/60'/1'/0/0"
path: "m/44'/60'/2'/0/0"
```

source: https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#user-content-Account

Reply:
thanks for this callout!